src/main/java/com/haxwell/apps/quizki/_servletFilters/CorsFilter.java

```diff
@@ -21,7 +21,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
 		HttpServletResponse response = (HttpServletResponse) servletResponse;
 		HttpServletRequest req = (HttpServletRequest)servletRequest;
 
-		response.setHeader("Access-Control-Allow-Origin", "http://localhost:8100"); // was *
+		response.setHeader("Access-Control-Allow-Origin", "*"); // was http://localhost:8100
```

Comment:
This defines where requests are allowed to come from.. it should be the URL that the local client is running on, I believe. * is too general.